config/initializers/shopify_app.rb

```diff
@@ -1,11 +1,12 @@
 ShopifyApp.configure do |config|
   config.application_name = "Triggerify"
-  config.api_key = ENV['SHOPIFY_CLIENT_API_KEY']
-  config.secret = ENV['SHOPIFY_CLIENT_API_SECRET']
   config.old_secret = ""
   config.scope = "write_orders, write_products, write_customers, write_fulfillments, write_themes"
   config.embedded_app = true
   config.after_authenticate_job = false
   config.api_version = "unstable"
   config.session_repository = Shop
+
+  config.api_key = ENV['SHOPIFY_CLIENT_API_KEY'].presence || raise("Missing SHOPIFY_CLIENT_API_KEY env variable")
+  config.secret = ENV['SHOPIFY_CLIENT_API_SECRET'].presence || raise("Missing SHOPIFY_CLIENT_API_SECRET env variable")
```

Comment:
I would have stuck to `ENV.fetch('...')` but whatever floats your boat.

Reply:
Could do that if we were to comment the placeholders in the env file.